app/jobs/notifications/milestone_job.rb

```diff
@@ -0,0 +1,12 @@
+module Notifications
+  class MilestoneJob < ApplicationJob
+    queue_as :send_milestone_notification
+
+    def perform(milestone_hash, service = Notifications::Milestone::Send)
+      article = Article.find_by(id: milestone_hash[article])
```

Comment:
Probably, here is a typo - ` milestone_hash[article]` instead of `milestone_hash[:article]` ? I also suggest using `article_id` instead of article for clarity.
And I'm concerned that this issue is not caught by test, so I suggest adding a test with actual article finding.

Reply:
Oups. A typo indeed. I added 2 tests (the .with). And I rename :article to :article_id